routes/search/router.js

```diff
@@ -0,0 +1,39 @@
+/** @module search/router */
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const middleware =  require('../middleware');
+const rootUrl = require("../../config").url;
+const utils = require('../utils');
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Types.ObjectId;
+const Freelance = mongoose.model('Freelance');
+
+// Supported methods
+router.all('/', middleware.supportedMethods('GET, OPTIONS'));
+
+// Return index
+router.get('/', function(req, res, next) {
+  const keyword = req.query.keyword;
```

Comment:
Later, we should split `keyword` on `" "` and create a filter for each word in the keyword.